### PR TITLE
Migrate experience level controller to seperate api project

### LINKED ISSUE #148: Migrate controllers to seperate API project
- [x] Create API project
- [x] Migrate Experience Level Controller
- [x] Migrate Listing Controller
- [x] Migrate Company Controller
- [x] Migrate Inquiry Controller
- [x] Migrate Auth Controller

### PR DESCRIPTION
**Pull request by OscarThorburn**
#148

### CODE REVIEW

File: src/GT.API/Controllers/ExperienceLevelController.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+			var experienceLevels = await _experienceLevelService.GetAllAsync();
+
+			if (experienceLevels == null)
+			{
+				return NotFound();
+			}
+
+			var result = JsonConvert.SerializeObject(experienceLevels);
+
+			return result;
```
Comment (antoncarlgren) [inline src/GT.API/Controllers/ExperienceLevelController.cs:33]: Should this return Ok(result); ?

File: src/GT.API/Controllers/ExperienceLevelController.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+			var experienceLevel = await _experienceLevelService.GetByIdAsync(id);
+
+			if (experienceLevel == null)
+			{
+				return NotFound();
+			}
+
+			var result = JsonConvert.SerializeObject(experienceLevel);
+
+			return result;
```
Comment (antoncarlgren) [inline src/GT.API/Controllers/ExperienceLevelController.cs:54]: Should this return Ok(result); ?

**Review by antoncarlgren — CHANGES_REQUESTED** [review]

File: src/GT.Core/Services/Impl/GTExperienceLevelService.cs [hunk 3]
```diff
@@ -108,5 +124,68 @@ public async Task<bool> ExistsByNameAsync(string name)
...
+				return experienceLevelDTO;
+			}
+			catch (Exception e)
+			{
+				_logger.LogError(e.Message);
+				return null;
+			}
+		}
+
+		public async Task UpdateAsync(ExperienceLevelDTO experienceLevelDTO, string name)
```
Comment (JohanSaisa) [inline src/GT.Core/Services/Impl/GTExperienceLevelService.cs:159]: Any specific reason we update by name and not id?

**Review by JohanSaisa — APPROVED** [review]
Approved. One comment for discussion.

File: src/GT.API/Controllers/ExperienceLevelController.cs (hunk 2 above)
Comment (OscarThorburn) [reply src/GT.API/Controllers/ExperienceLevelController.cs:54]: Fixed

File: src/GT.API/Controllers/ExperienceLevelController.cs (hunk 1 above)
Comment (OscarThorburn) [reply src/GT.API/Controllers/ExperienceLevelController.cs:33]: Fixed

File: src/GT.Core/Services/Impl/GTExperienceLevelService.cs (hunk 3 above)
Comment (OscarThorburn) [reply src/GT.Core/Services/Impl/GTExperienceLevelService.cs:159]: This was a legacy from old code when the service was built. PO have created a card to go through all services regarding this in the backlog

**Review by RasmusBroborg — APPROVED** [review]

**Review by antoncarlgren — APPROVED** [review]

**Pull request merged**